vmngclient/session.py

```
@@ -107,6 +110,14 @@ def create_vManageSession(
     return session
 
 
+def check_vmanage_server_connection(url):
+    url = f"http://{url}"
+    try:
+        head(url, timeout=2)
+    except ConnectionError:
+        logger.error("vManage server is not available")
```

Should we reraise an exception?

---

Exception is being reraised now